src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  eventElement.preventDefault();
+
+  const largePrevious = document.querySelector('.gallery__large-img');
+  largePrevious.src = eventElement.target.src;
+  
+};
+
+gallery.addEventListener('click', changeImage);
```

Comment:
The click event is attached to the whole gallery, which includes the large image and the container itself. This means that if a user clicks on the large image or the gallery container (but not on a thumbnail), the `changeImage` function will still be triggered, which is not the intended behavior. You should add a condition to check if the clicked element is a thumbnail before changing the source of the large image.